dashboard/src/features/transactions/components/New/FormActionItem.jsx

```diff
@@ -7,15 +7,17 @@ const SPEND_ACCOUNT_KEY = 'spend_account'
 const SPEND_UNSPENT_KEY = 'spend_account_unspent_output'
 const CONTROL_ACCOUNT_KEY = 'control_account'
 const CONTROL_PROGRAM_KEY = 'control_program'
-const RETIRE_ASSET_KEY = 'retire_asset'
+const CONTROL_RECEIVER_KEY = 'control_receiver'
+const RETIRE_ASSET_KEY = 'retire'
```

Comment:
This is okay to sneak in here, but probably better to split it off into a separate PR in the future.